lib/browser/api/dialog.js

```diff
@@ -86,7 +95,7 @@ module.exports = {
     wrappedCallback = typeof callback === 'function' ? function (success, result) {
       return callback(success ? result : void 0)
     } : null
-    return binding.showOpenDialog(String(options.title), String(options.defaultPath), options.filters, properties, window, wrappedCallback)
+    return binding.showOpenDialog(String(options.title), String(options.buttonLabel), String(options.defaultPath), options.filters, properties, window, wrappedCallback)
```

Comment:
I think we can remove the `String(...)` conversion now, since the code is checking each parameter's type.